2015-12-07  Aurelien Jarno  <aurelien@aurel32.net>

	* sysdeps/unix/sysv/linux/alpha/bits/termios.h (IXANY, IUCLC,
	IMAXBEL, IUTF8): Make always visible since they're needed by POSIX.
	(NL2, NL3): Guard BSD terminal modes definitions with [__USE_MISC].

--- a/sysdeps/unix/sysv/linux/alpha/bits/termios.h
+++ b/sysdeps/unix/sysv/linux/alpha/bits/termios.h
@@ -70,13 +70,10 @@ struct termios
 #define ICRNL	0000400
 #define IXON	0001000
 #define IXOFF	0002000
-#ifdef __USE_MISC
-  /* POSIX.1 doesn't want these... */
-# define IXANY		0004000
-# define IUCLC		0010000
-# define IMAXBEL	0020000
-# define IUTF8		0040000
-#endif
+#define IXANY	0004000
+#define IUCLC	0010000
+#define IMAXBEL	0020000
+#define IUTF8	0040000
 
 /* c_oflag bits */
 #define OPOST	0000001
@@ -93,8 +90,10 @@ struct termios
 # define NLDLY	00001400
 # define   NL0	00000000
 # define   NL1	00000400
-# define   NL2	00001000
-# define   NL3	00001400
+# if defined __USE_MISC
+#  define   NL2	00001000
+#  define   NL3	00001400
+# endif
 # define TABDLY	00006000
 # define   TAB0	00000000
 # define   TAB1	00002000
